src/main/java/edu/ksu/canvas/attendance/exception/AttendanceAssignmentException.java

```diff
@@ -0,0 +1,46 @@
+package edu.ksu.canvas.attendance.exception;
+
+import edu.ksu.canvas.attendance.entity.AttendanceAssignment;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'edu.ksu.canvas.attendance.entity.AttendanceAssignment'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)